### PR TITLE
Add Cleaning Cost

### PR DESCRIPTION
**Pull request by hamzaehsan95**

### CODE REVIEW

File: app/src/main/res/layout/fragment_cleaning_cost.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        android:layout_marginBottom="@dimen/margin_all"
+        android:layout_marginEnd="@dimen/margin_all"
+        android:layout_marginLeft="@dimen/margin_all"
+        android:layout_marginRight="@dimen/margin_all"
+        android:layout_marginStart="@dimen/margin_all"
+        android:visibility="gone"
+        android:layout_marginTop="@dimen/margin_all"
+        android:textColor="#000"
+        android:textSize="20sp"
+        android:text="No Accomodations Found!"
```
Comment (Muhammad-Babar) [inline app/src/main/res/layout/fragment_cleaning_cost.xml]: Please don't use strings anywhere in the project. Define them in the strings file and then use them via their references.

File: app/src/main/java/co/thedevden/nice2staycrm/fragment/CleaningCostFragment.java [hunk 2]
```diff
@@ -20,6 +59,60 @@ public int returnLayout() {
...
+            @Override
+            protected void response(retrofit2.Response response, View view) {
+
+                cleaningCostList = (List<CleaningCostModel>) response.body();
+
+                if (cleaningCostList.size() > 0)
+                    showLayout(cleaningCostList);
+                else
+                    noCleaningCosts();
+            }
```
Comment (Muhammad-Babar) [inline app/src/main/java/co/thedevden/nice2staycrm/fragment/CleaningCostFragment.java]: cleaningCostList = (List<CleaningCostModel>) response.body();
                 if (cleaningCostList.size() > 0)
                    showLayout(cleaningCostList);
                else
                    noCleaningCosts();
------------------
you have used 7 lines. you can do same work in 3 lines.
                cleaningCostList = (List<CleaningCostModel>) response.body();
                 if (cleaningCostList.size() > 0) showLayout(cleaningCostList);
                else noCleaningCosts();


try to reduce number of lines

File: app/src/main/res/layout/fragment_cleaning_cost.xml [hunk 3]
```diff
@@ -56,7 +56,7 @@
         android:layout_marginTop="@dimen/margin_all"
         android:textColor="#000"
         android:textSize="20sp"
-        android:text="No Accomodations Found!"
+        android:text="No Cleaning Cost Found!"
```
Comment (Muhammad-Babar) [inline app/src/main/res/layout/fragment_cleaning_cost.xml]: Don't use strings here, please.

File: app/src/main/res/layout/cleaning_cost_list_item.xml [hunk 4]
```diff
@@ -139,6 +139,23 @@
...
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal"
+                    android:gravity="right"
+                    android:layout_marginTop="@dimen/fab_margin">
+
+                    <ImageButton
+                        android:id="@+id/edit_cleaningCostt_image"
+                        android:layout_width="40dp"
+                        android:layout_height="40dp"
```
Comment (Muhammad-Babar) [inline app/src/main/res/layout/cleaning_cost_list_item.xml]: Do you want fix size for all screen sizes?

Comment (hamzaehsan95) [reply app/src/main/res/layout/cleaning_cost_list_item.xml]: no i want for all screen sizes. Should I use weight here?

**Pull request merged**